vertica/pyproject.toml

```
@@ -29,7 +29,7 @@ classifiers = [
     "Topic :: System :: Monitoring",
 ]
 dependencies = [
-    "datadog-checks-base>=25.1.0",
+    "datadog-checks-base>=25.7.0",
```

```suggestion
    "datadog-checks-base>=26.0.0",
```
The current CI errors should resolve after releasing the datadog checks base [v26.0.0](https://github.com/DataDog/integrations-core/pull/12571) and updating the version here